File: elixir/7ku/my_language_skills.ex
defmodule MyLanguages do
  def my_languages(results) do
    results
    |> Enum.filter(fn {_x, y} -> y >= 60 end)
    |> Enum.sort_by(fn {_x, y} -> y end, :desc)
    |> Enum.map(fn {x, _y} -> x end)
  end
end

File: ruby/6ku/My Language Skills.rb
# frozen_string_literal: true
# My Language Skills
# https://www.codewars.com/kata/5b16490986b6d336c900007d/train/ruby

def my_languages(results)
  results.select { |k, v| v >= 60 }.sort_by { |_k, v| -v }.map(&:first)
end

p my_languages({"Java" => 10, "Ruby" => 80, "Python" => 65})# ["Ruby", "Python"]
